skinStyles/extensions/SyntaxHighlight_GeSHi/ext.pygments.less

```diff
@@ -305,6 +305,8 @@
 
 		.linenos {
 			color: var( --color-subtle );
+			background-color: var( --color-surface-2 );
+			border-left: var( --border-width-base ) solid var( --border-color-base );
```

Comment:
You can just use `var( --border-base )` here

Reply:
Instead of adding a new border, perhaps we should do `margin-left: 1px` instead so that we keep the original border from the code block